### ISSUE TITLE
Only shows in multi-instance mode

### ISSUE THREAD

**Issue by rmm2908** [ziyagenc/phi#8]
The icon will only show in the top bar when multi-instance mode is switched on. Then I can see stats, regardless of whether a second instance is configured. Used to work on Gnome 46. 
OS: Fedora 41
Gnome: 47
Extension: v1.6

**Comment by rmm2908** [comment]
Also I got the following error when monitoring gnome logs:
```
Error: Wrong type undefined; string expected
set text2@file:///home/ryan/.local/share/gnome-shell/extensions/phi@ziyagenc.github.com/mitems.js:44:7
_updateUI@file:///home/ryan/.local/share/gnome-shell/extensions/phi@ziyagenc.github.com/mpihole.js:219:9
async*_startUpdating/this._fetchTimeoutId<@file:///home/ryan/.local/share/gnome-shell/extensions/phi@ziyagenc.github.com/mpihole.js:239:16
@resource:///org/gnome/shell/ui/init.js:21:20
   Error in _updateUI
```

**Comment by rmm2908** [comment]
FYI, I corrected the error message above, since the original one was because I was in multi-instance mode, the new one is when switching to single instance and I think more relevant

**Comment by ziyagenc** [comment]
It seems that with GNOME 47, context property of StatsItem creates a conflict. With GNOME 47, GNOME Shell expects a ClutterContext object at context, but Phi stores an ItemContext which is used to track if there is an error while fetching the statistics in Single Instance Mode. MultiStatItem does not have a context property, which is why this issue does not arise in Multi Instance Mode. 

The issue is fixed in 8dda09c. The new version should be available soon in the GNOME Shell Extensions page.

Thanks for opening this issue. If it persist, please do not hesitate to let me know.

**Comment by rmm2908** [comment]
Thanks, I will try it when I get home and then close the issue.
